src/Pages/Media/style.ts

```diff
@@ -1,34 +1,173 @@
 import { Carousel } from "react-bootstrap";
 import styled from "styled-components";
 
+export const sizes = {
+  mobileES: "100px",
+  mobileS: "375px",
+  mobileM: "425px",
+  mobileL: "576px",
+  tablet: "768px",
+  laptop: "1024px",
+  laptopL: "1440px",
+  desktop: "2560px",
+};
+
+export const devices = {
+  mobileES: `min-width: ${sizes.mobileES}`,
+  mobileS: `min-width: ${sizes.mobileS}`,
+  mobileM: `min-width: ${sizes.mobileM}`,
+  mobileL: `min-width: ${sizes.mobileL}`,
+  tablet: `min-width: ${sizes.tablet}`,
+  laptop: `min-width: ${sizes.laptop}`,
+  laptopL: `min-width: ${sizes.laptopL}`,
+  desktop: `min-width: ${sizes.desktop}`,
+};
+
```

Comment:
You don't need to re-define for all the pages. Reuse the ones from other pages

Reply:
Or create a file called `arc/style.ts` and add this there and import this file wherever required